packages/aws-cdk-lib/aws-cognito/README.md

```diff
@@ -1002,3 +1002,24 @@ const userpool = new cognito.UserPool(this, 'UserPool', {
 ```
 
 By default deletion protection is disabled.
+
+### User Pool Group
+
+Support for groups in Amazon Cognito user pools enables you to create and manage groups, add users to groups, and remove users from groups.
+Use groups to create collections of users to manage their permissions or to represent different types of users.
+
+You can assign an AWS Identity and Access Management (IAM) role to a group to define the permissions for members of a group.
+
+For more information, see [Adding groups to a user pool](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-user-groups.html).
+
+```ts
```

Comment:
Users will also want to know how to use the following.
```ts
userPool.addGroup('another-group', {
  precedence: 1,
});
```

Reply:
Thanks.
I've updated docs.